src/Selectors.ts

```diff
@@ -219,18 +220,20 @@ class Selectors {
     },
   );
 
-  getLocationCircles: PropsSelector<LocationCircle[]> = createSelector([getLocationFeatures], locations =>
-    _.flatMap(locations, location => [
-      {
+  getLocationCircles: PropsSelector<LocationCircle[]> = createSelector([getLocationFeatures], locations => {
+    const circles = [];
```

Comment:
is this more performant that _.flatMap?

Reply:
I don't remember why I changed this here :)  Why do you ask?

This PR is too old now and needs to be reviewed. We actually got rid of lodash in the current version.

Reply:
i'm interested in developing and adding some functionality, thought that trawling the open issues and PRs might be a good place to start. I'm a big fan of FP concepts like flatMap so was just interested if there was a reason to go back to a = [], a.push_back.

Reply:
I replied in #72. My main motivation for this PR was to make it possible to use flowmap.gl in [Observable](https://observablehq.com/). However, the fact that luma.gl didn't have a UMD build was an obstacle and I gave up.